recipe/meta.yaml

```diff
@@ -20,12 +20,16 @@ requirements:
     - autoconf
     - automake
   host:
+{% if not compiler('c').startswith("toolchain") %}
+    - llvm-openmp  # [osx]
+{% else %}
     - openmp 4.0.*  # [osx]
+{% endif %}
     - fftw
     - cunit
     - ncurses
   run:
-    - openmp 4.0.*  # [osx]
+    - llvm-openmp  # [osx]
```

Comment:
Actually this part doesn't make sense. Wouldn't you still need `openmp` in `run`?

Reply:
yes. this was a mistake. The tests suite is run at build time so this wasn't caught by the CI. 
One option would be to attempt a build at test time using a minimal file such as the one in this homebrew recipe:
https://github.com/brewsci/homebrew-science/blob/master/Formula/nfft.rb